frontend/src/app/Pagination.tsx

```diff
@@ -1,65 +1,128 @@
 import React, { Fragment } from 'react';
-import { Button, Badge, Level, LevelItem } from '@patternfly/react-core';
+import { Level, LevelItem } from '@patternfly/react-core';
+import { ContextSelector, ContextSelectorItem } from '@patternfly/react-core';
+import { Dropdown } from '@app/Dropdown';
 
 export interface IProps {
   handleMoveLeft: () => any
   handleMoveRight: () => any
+  handleMoveToFirst: () => any
+  handlePerPageLimit: (itemsPerPage) => any
   pageNumber: number
   nextPageRecordCount: number
-  noOfRecordsOnPage: number
+  perPageLimit:number
+  showOptions: boolean
+  bottom: boolean
 }
 
 class Pagination extends React.Component<IProps> {
 
+  public dropdownItems = [
+    '25 items per page',
+    '50 items per page',
+    '75 items per page',
+    '100 items per page'
+    ];
+
+  public state = {
+    filteredItems: this.dropdownItems,
+    isExpanded: false, 
+    isOpen: false,
+    itemsPerPage: 25,
+    renderSearch: false,
+    searchValue: '',
+    selected: this.dropdownItems[0]
+
+  };
+
+  public button1 = true;
+  public button2 = true;
```

Comment:
These button1, button2, button3 should be renamed to something more significant.

Reply:
Yes Joseph!
I am changing button1 to firstPageButton, button2 to previousPageButton and button3 to nextPageButton as per their functionality

Reply:
sounds good.